code-gen-projects/schema/nested_struct.isl

```diff
@@ -5,7 +5,8 @@ type::{
     B: int,
     C: {
         fields: {
-            D: bool
+            D: bool,
+            E: { element: int } // default sequence type is `list`
```

Comment:
FYI—it's okay to require things to be explicit. We can always relax that requirement later if we want, but if we start out with implicit list, we can't take that back.

This is one case, however, where I'm 99% sure that it's fine for `list` to be the default sequence.

Reply:
Makes sense. I will update this to require having a `type` constraint.

Reply:
Updated. Added tests for it in `code-gen-tests` module.